nominees/document-gps.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Document GPS",
+  "aliases": [
+    ""
+  ],
+  "description": "A secure way to transmit information using Web3 protocols while also reducing large carbon emissions from digital infrastructure.",
+  "website": "https://shelterzoom.com/document-gps",
+  "license": [
+    {
+      "spdx": "CC-BY-1.0",
+      "licenseURL": "https://creativecommons.org/licenses/"
```

Comment:
The licenseURL provided should explicitly indicate the CC-BY-1.0 license. Thanks